src/main/java/com/google/devtools/build/lib/rules/java/JavaInfo.java

```diff
@@ -463,14 +463,29 @@ public SkylarkNestedSet getTransitiveRuntimeJars() {
 
   @SkylarkCallable(
       name = "transitive_compile_time_jars",
-      doc = "Depset of compile time jars recusrively required by this target. See `compile_jars` "
+      doc = "Depset of compile time jars recursively required by this target. See `compile_jars` "
           + "for more details.",
       structField = true
   )
   public SkylarkNestedSet getTransitiveCompileTimeJars() {
     return SkylarkNestedSet.of(Artifact.class, getTransitiveDeps());
   }
 
+  @SkylarkCallable(
+      name = "transitive_full_compile_time_jars",
```

Comment:
The name is a bit of a mouthful, but it mirrors the existing `transitive_compile_time_jars`. I noticed it's also exposed as `transitive_deps`, maybe this should just be `transitive_full_deps`?